kafka/src/main/java/io/micronaut/configuration/kafka/NoopProducer.java

```diff
@@ -0,0 +1,112 @@
+/*
```

Comment:
Why is this needed? Is it so that `@KafkaClient`s don't fail when Kafka is disabled?